bool is_human_detected
float32 distance